README.md

```diff
@@ -14,110 +14,10 @@ Additionally, there are [several tools](https://maplibre.org/martin/tools.html)
-martin postgres://... postgres://...  --save-config config.yaml
-
-# use configuration file instead of auto-discovery
-martin --config config.yaml
-```
-
-#### Docker Example
-
-_See [Docker instructions](https://maplibre.org/martin/run-with-docker.html) in the Martin book._
-
-Martin is also available as a [Docker image](https://ghcr.io/maplibre/martin). You could either share a configuration file from the host with the container via the `-v` param, or you can let Martin auto-discover all sources e.g. by passing `DATABASE_URL` or specifying the .mbtiles/.pmtiles files.
-
-```shell
-export PGPASSWORD=postgres  # secret!
-docker run -p 3000:3000 \
-           -e PGPASSWORD \
-           -e DATABASE_URL=postgresql://user@host:port/db \
-           -v /path/to/config/dir:/config \
-           ghcr.io/maplibre/martin --config /config/config.yaml
-```
-
-## API
-
-_See [API documentation](https://maplibre.org/martin/using.html) in the Martin book._
-
-Martin data is available via the HTTP `GET` endpoints:
-
-| URL                                     | Description                                   |
-|-----------------------------------------|-----------------------------------------------|
-| `/`                                     | Status text, that will eventually show web UI |
-| `/catalog`                              | List of all sources                           |
-| `/{sourceID}`                           | Source TileJSON                               |
-| `/{sourceID}/{z}/{x}/{y}`               | Map Tiles                                     |
-| `/{source1},…,{sourceN}`                | Composite Source TileJSON                     |
-| `/{source1},…,{sourceN}/{z}/{x}/{y}`    | Composite Source Tiles                        |
-| `/sprite/{spriteID}[@2x].{json,png}`    | Sprites (low and high DPI, index/png)         |
-| `/font/{font}/{start}-{end}`            | Font source                                   |
-| `/font/{font1},…,{fontN}/{start}-{end}` | Composite Font source                         |
-| `/health`                               | Martin server health check: returns 200 `OK`  |
-
-## Re-use Martin as a library
-
-Martin can be used as a standalone server, or as a library in your own Rust application. When used as a library, you can use the following features:
-
-* **postgres** - enable PostgreSQL/PostGIS tile sources
-* **pmtiles** - enable PMTile tile sources
-* **mbtiles** - enable MBTile tile sources
-* **fonts** - enable font sources
-* **sprites** - enable sprite sources
-
-## Documentation
-
-See [Martin book](https://maplibre.org/martin/) for complete documentation.
+* [Installation](https://maplibre.org/martin/installation.html)
+* Running with [CLI](https://maplibre.org/martin/run-with-cli.html) or [configuration file](https://maplibre.org/martin/config-file.html)
+* [Usage and API](https://maplibre.org/martin/using.html)
+<!-- * [Quick Start](https://maplibre.org/martin/quick-Start.html) -->
```

Comment:
To bypass the markdwon link check, this line is removed temporally. Will be added after this get merged.